config/locales/idv/en.yml

```diff
@@ -33,19 +33,18 @@ en:
       unsupported_jurisdiction: Sorry, we can't verify people from this state.
       unsupported_jurisdiction_sp: Please visit %{sp_name} to access your account.
     form:
-      activate_by_mail: get a code by letter.
+      activate_by_mail: get a code via mail instead ›
```

Comment:
I would consider inserting the > via CSS using :after rather than having it be part of the text string. I think it will be better for a11y, and also be more flexible if the preferred style were to change.

Reply:
not opposed to it in concept, however, there is at least one other place where the guillemet is present and I was just going with the flow.

Originally, the one in English was different than Spanish and French, but per some the comment [here](https://github.com/18F/identity-idp/pull/2194/files/5888810b63228d76798d95d31721351edfc8d2e7#diff-c62b6e9e48a9b41dcf4b8772eb40d9d9), i changed them to be the same.